src/components/DayPickerRangeController.jsx

```diff
@@ -301,15 +305,14 @@ export default class DayPickerRangeController extends React.PureComponent {
         initialVisibleMonth !== prevInitialVisibleMonth
         && !prevFocusedInput
         && didFocusChange
+      ) || (
+        nextVisibleMonth
+        && moment.isMoment(nextVisibleMonth)
```

Comment:
if it's truthy and not a moment object, i feel like this should throw, instead of just silently ignoring it